File: Python/Lab_Data_Types_and_Variables/CenturiesToMinutes.py
centuries = float(input())

years = centuries * 100
days = years * 365.2422
hours = years * 24
minutes = hours * 60

print(f'{centuries:.0f} centuries = {years:.0f} years = {days:.0f} days = {hours:.0f} hours = {minutes:.0f} minutes')

File: JavaScript/Data Types and Variables/Lab/centuriesToMinutes.js
function centuriesMinute(input) {
    let centuries = Number(input);
    let years = centuries * 100;
    let days = Math.floor(years * 365.2422);
    let hours = days * 24;
    let minutes = hours * 60;

    console.log(`${centuries} centuries = ${years} years = ${days} days = ${hours} hours = ${minutes} minutes`);
}

centuriesMinute(1);
centuriesMinute(5);